test/e2e/e2e_test.go

```diff
@@ -1424,6 +1429,11 @@ func initPodEvictorOrFail(t *testing.T, clientSet clientset.Interface, getPodsAs
 	if err != nil || len(evictionPolicyGroupVersion) == 0 {
 		t.Fatalf("Error creating eviction policy group: %v", err)
 	}
+
+	ctx := context.Background()
+	eventBroadcaster, eventRecorder := utils.GetRecorderAndBroadcaster(ctx, clientSet)
+	defer eventBroadcaster.Shutdown()
```

Comment:
`defer eventBroadcaster.Shutdown()` needs to be invoked outside of `initPodEvictorOrFail`

Reply:
I tried removing all defer shutdowns from e2e tests just to quickly to see if it would make e2e pass, but it didn't :(

I will make that change anyways :)

Reply:
Hmm, maybe I imagined that, one of the e2e passed now